.travis.yml

```diff
@@ -43,6 +43,7 @@ language: c
 ###############################################################################
 before_install:
     - sudo apt-get update
+    - sudo apt-get install splint
```

Comment:
You are right. I should have added. I merged this PR but when I edit this file, I will add comments.